Bindings/Java/OpenSimJNI/Test/testContext.cpp

```diff
@@ -176,6 +176,14 @@ int main()
     assert(context->getLocked(dr_elbow_flexNew));
     ASSERT_EQUAL(0.5, context->getValue(dr_elbow_flexNew), 0.000001);
 
+    // Exercise Editing workflow
+    OpenSim::Body& bdy = model->updBodySet().get("r_humerus");
+    AbstractProperty& massProp = bdy.updPropertyByName("mass");
```

Comment:
Is there a reason for using the property interface here instead `body.setMass(body.getMass()+1.0)`? Seems harder to read. But is this is actually what the GUI executes when an edit is made, it makes sense to leave it as is. Can you make a note of that, if that's the case?